jf_agent/data_manifests/git/adapters/github.py

```diff
@@ -36,18 +33,10 @@ def __init__(
         # Session fields
         self.token = token
 
-        if base_url and 'api/v3' in base_url:
-            # Github server clients provide an API with a trailing '/api/v3'
-            # replace this with the graphql endpoint
-            self.base_url = base_url.replace('api/v3', 'api/graphql')
-        else:
-            self.base_url = 'https://api.github.com/graphql'
+        # Translate base URL if we are using cloud or server
```

Comment:
This looks like it's totally swapping over to the new method, are we doing that or feature flagging?

Reply:
This is a little confusing, but this is the `data_manifests` gql adapter and not the one that the actual data processing uses. This has always used GQL so we're not hiding it behind the feature flag at all

Reply:
There should be an effort to join this with the other `github_gql` adapter, but I decided to leave the seperately as we roll out the new feature. Ideally we'd consolidate them when we move things to the `jf_ingest` repository, or sometime before that

Reply:
Ah yeah makes sense. missed that.